services/src/main/java/org/jboss/windup/web/services/rest/RegisteredApplicationEndpointImpl.java

```diff
@@ -36,23 +47,112 @@
     }
 
     @Override
-    public RegisteredApplication registerApplication(RegisteredApplication application)
+    public RegisteredApplication getApplication(int id) {
```

Comment:
Sorry about that, I already use it, I just created this file before configuring Idea. I didn't notice.